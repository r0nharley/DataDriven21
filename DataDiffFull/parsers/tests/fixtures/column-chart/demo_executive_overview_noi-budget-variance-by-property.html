<svg version="1.1" class="highcharts-root" style="font-family:&quot;Lucida Grande&quot;, &quot;Lucida Sans Unicode&quot;, Arial, Helvetica, sans-serif;font-size:12px;" xmlns="http://www.w3.org/2000/svg" width="1293" height="343">
    <desc>Created with Highstock 5.0.6</desc>
    <defs>
        <clipPath id="highcharts-w9nkk1v-10">
            <rect x="0" y="0" width="1217" height="220" fill="none"></rect>
        </clipPath>
        <clipPath id="highcharts-w9nkk1v-11">
            <rect x="0" y="0" width="1217" height="220" fill="none"></rect>
        </clipPath>
    </defs>
    <rect fill="#ffffff" class="highcharts-background" x="0" y="0" width="1293" height="343" rx="0" ry="0"></rect>
    <rect fill="none" class="highcharts-plot-background" x="66" y="10" width="1217" height="220"></rect>
    <g class="highcharts-grid highcharts-xaxis-grid ">
        <path fill="none" class="highcharts-grid-line" d="M 114.5 10 L 114.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 162.5 10 L 162.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 211.5 10 L 211.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 260.5 10 L 260.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 308.5 10 L 308.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 357.5 10 L 357.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 406.5 10 L 406.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 454.5 10 L 454.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 503.5 10 L 503.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 552.5 10 L 552.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 600.5 10 L 600.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 649.5 10 L 649.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 698.5 10 L 698.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 747.5 10 L 747.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 795.5 10 L 795.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 844.5 10 L 844.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 893.5 10 L 893.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 941.5 10 L 941.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 990.5 10 L 990.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1039.5 10 L 1039.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1087.5 10 L 1087.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1136.5 10 L 1136.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1185.5 10 L 1185.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1233.5 10 L 1233.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 1282.5 10 L 1282.5 230" opacity="1"></path>
        <path fill="none" class="highcharts-grid-line" d="M 65.5 10 L 65.5 230" opacity="1"></path>
    </g>
    <g class="highcharts-grid highcharts-yaxis-grid ">
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 230.5 L 1283 230.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 175.5 L 1283 175.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 120.5 L 1283 120.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 65.5 L 1283 65.5" opacity="1"></path>
        <path fill="none" stroke="#D8D8D8" stroke-width="1" stroke-dasharray="1,3" class="highcharts-grid-line" d="M 66 9.5 L 1283 9.5" opacity="1"></path>
    </g>
    <rect fill="none" class="highcharts-plot-border" x="66" y="10" width="1217" height="220"></rect>
    <g class="highcharts-axis highcharts-xaxis ">
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 114.5 230 L 114.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 162.5 230 L 162.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 211.5 230 L 211.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 260.5 230 L 260.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 308.5 230 L 308.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 357.5 230 L 357.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 406.5 230 L 406.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 454.5 230 L 454.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 503.5 230 L 503.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 552.5 230 L 552.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 600.5 230 L 600.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 649.5 230 L 649.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 698.5 230 L 698.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 747.5 230 L 747.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 795.5 230 L 795.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 844.5 230 L 844.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 893.5 230 L 893.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 941.5 230 L 941.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 990.5 230 L 990.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1039.5 230 L 1039.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1087.5 230 L 1087.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1136.5 230 L 1136.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1185.5 230 L 1185.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1233.5 230 L 1233.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 1283.5 230 L 1283.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-tick" stroke="#ccd6eb" stroke-width="1" d="M 65.5 230 L 65.5 240" opacity="1"></path>
        <path fill="none" class="highcharts-axis-line" stroke="#ccd6eb" stroke-width="1" d="M 66 230.5 L 1283 230.5"></path>
    </g>
    <g class="highcharts-axis highcharts-yaxis ">
        <path fill="none" class="highcharts-axis-line" d="M 66 10 L 66 230"></path>
    </g>
    <g class="highcharts-series-group">
        <g class="highcharts-series highcharts-series-0 highcharts-column-series highcharts-color-undefined highcharts-tracker " transform="translate(66,10) scale(1 1)" width="220" height="1217" clip-path="url(https://staging.rentlytics.com/demo/bi/overview-executive/visualization#highcharts-w9nkk1v-11)">
            <rect x="12.5" y="110.5" width="24" height="31" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="60.5" y="110.5" width="24" height="44" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="109.5" y="104.5" width="24" height="6" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="158.5" y="40.5" width="24" height="70" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="206.5" y="110.5" width="24" height="26" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="255.5" y="101.5" width="24" height="9" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="304.5" y="110.5" width="24" height="57" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="352.5" y="73.5" width="24" height="37" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="401.5" y="110.5" width="24" height="16" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="450.5" y="109.5" width="24" height="1" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="498.5" y="102.5" width="24" height="8" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="547.5" y="110.5" width="24" height="41" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="596.5" y="110.5" width="24" height="24" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="644.5" y="69.5" width="24" height="41" fill="#6eda55" class="highcharts-point " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="693.5" y="110.5" width="24" height="38" fill="#fc7570" class="highcharts-point highcharts-negative " stroke="#ffffff" stroke-width="1"></rect>
            <rect x="742.5" y="102.5" width="24" height="8" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="791.5" y="110.5" width="24" height="25" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="839.5" y="110.5" width="24" height="14" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="888.5" y="110.5" width="24" height="6" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="937.5" y="110.5" width="24" height="33" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="985.5" y="105.5" width="24" height="5" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="1034.5" y="102.5" width="24" height="8" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="1083.5" y="82.5" width="24" height="28" fill="#6eda55" class="highcharts-point" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="1131.5" y="110.5" width="24" height="33" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
            <rect x="1180.5" y="110.5" width="24" height="43" fill="#fc7570" class="highcharts-point highcharts-negative" stroke="#ffffff" stroke-width="1"></rect>
        </g>
        <g class="highcharts-markers highcharts-series-0 highcharts-column-series highcharts-color-undefined " transform="translate(66,10) scale(1 1)" width="220" height="1217" clip-path="none"></g>
    </g>
    <g class="highcharts-axis-labels highcharts-xaxis-labels ">
        <text x="92.93272486435067" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 92.93272486435067 246)" y="246" opacity="1">
            <tspan>Alamo Place</tspan>
        </text>
        <text x="141.6127248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 141.6127248643507 246)" y="246" opacity="1">
            <tspan>Aspen Ridge</tspan>
        </text>
        <text x="190.29272486435067" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 190.29272486435067 246)" y="246" opacity="1">
            <tspan>Big Bear</tspan>
        </text>
        <text x="238.97272486435068" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 238.97272486435068 246)" y="246" opacity="1">
            <tspan>Bodega Valley Apts.</tspan>
        </text>
        <text x="287.65272486435066" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 287.65272486435066 246)" y="246" opacity="1">
            <tspan>China Camp</tspan>
        </text>
        <text x="336.33272486435067" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 336.33272486435067 246)" y="246" opacity="1">
            <tspan>Delfina</tspan>
        </text>
        <text x="385.0127248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 385.0127248643507 246)" y="246" opacity="1">
            <tspan>East Bay Crossing</tspan>
        </text>
        <text x="433.6927248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 433.6927248643507 246)" y="246" opacity="1">
            <tspan>Echo Park</tspan>
        </text>
        <text x="482.37272486435063" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 482.37272486435063 246)" y="246" opacity="1">
            <tspan>Gerry Trace</tspan>
        </text>
        <text x="531.0527248643506" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 531.0527248643506 246)" y="246" opacity="1">
            <tspan>Hanger 1</tspan>
        </text>
        <text x="579.7327248643506" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 579.7327248643506 246)" y="246" opacity="1">
            <tspan>Indian Valley</tspan>
        </text>
        <text x="628.4127248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 628.4127248643507 246)" y="246" opacity="1">
            <tspan>Joshua Tree</tspan>
        </text>
        <text x="677.0927248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 677.0927248643507 246)" y="246" opacity="1">
            <tspan>Juniper</tspan>
        </text>
        <text x="725.7727248643506" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 725.7727248643506 246)" y="246" opacity="1">
            <tspan>Lower Haight</tspan>
        </text>
        <text x="774.4527248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 774.4527248643507 246)" y="246" opacity="1">
            <tspan>Mackinac Landing</tspan>
        </text>
        <text x="823.1327248643506" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 823.1327248643506 246)" y="246" opacity="1">
            <tspan>Marina</tspan>
        </text>
        <text x="871.8127248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 871.8127248643507 246)" y="246" opacity="1">
            <tspan>Market Street Apts.</tspan>
        </text>
        <text x="920.4927248643506" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 920.4927248643506 246)" y="246" opacity="1">
            <tspan>North Beach</tspan>
        </text>
        <text x="969.1727248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 969.1727248643507 246)" y="246" opacity="1">
            <tspan>Papalote</tspan>
        </text>
        <text x="1017.8527248643505" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 1017.8527248643505 246)" y="246" opacity="1">
            <tspan>Potrero Hill</tspan>
        </text>
        <text x="1066.5327248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 1066.5327248643507 246)" y="246" opacity="1">
            <tspan>River Oaks</tspan>
        </text>
        <text x="1115.2127248643505" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 1115.2127248643505 246)" y="246" opacity="1">
            <tspan>The Baltimore</tspan>
        </text>
        <text x="1163.8927248643506" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 1163.8927248643506 246)" y="246" opacity="1">
            <tspan>The El Dorado</tspan>
        </text>
        <text x="1212.5727248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 1212.5727248643507 246)" y="246" opacity="1">
            <tspan>Travis Village</tspan>
        </text>
        <text x="1261.2527248643507" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:343px;text-overflow:ellipsis;" text-anchor="end" transform="translate(0,0) rotate(-45 1261.2527248643507 246)" y="246" opacity="1">
            <tspan>Viera Hills</tspan>
        </text>
    </g>
    <g class="highcharts-axis-labels highcharts-yaxis-labels ">
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:417px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="180" opacity="1">
            <tspan>-10.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:417px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="125" opacity="1">
            <tspan>0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:417px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="69" opacity="1">
            <tspan>10.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:417px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="235" opacity="1">
            <tspan>-20.0%</tspan>
        </text>
        <text x="51" style="color:#666666;cursor:default;font-size:11px;fill:#666666;width:417px;text-overflow:clip;" text-anchor="end" transform="translate(0,0)" y="15" opacity="1">
            <tspan>20.0%</tspan>
        </text>
    </g>
    <rect x="645" y="10" width="302" height="220" fill="rgba(51,92,173,0.25)" class="highcharts-selection-marker"></rect>
    <g class="highcharts-label highcharts-tooltip highcharts-color-undefined" style="cursor:default;pointer-events:none;white-space:nowrap;" transform="translate(633,-9999)" opacity="0" visibility="visible">
        <path fill="rgba(255, 255, 255, 0)" class="highcharts-label-box highcharts-tooltip-box" d="M 0 0 L 84 0 90 -6 96 0 180 0 C 180 0 180 0 180 0 L 180 102 C 180 102 180 102 180 102 L 0 102 C 0 102 0 102 0 102 L 0 0 C 0 0 0 0 0 0"></path>
    </g>
</svg>