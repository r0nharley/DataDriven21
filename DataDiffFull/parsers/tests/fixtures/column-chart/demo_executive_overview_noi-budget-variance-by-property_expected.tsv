Alamo Place	Data	-5.63636363636	#fc7570
Aspen Ridge	Data	-8.0	#fc7570
Big Bear	Data	1.09090909091	#6eda55
Bodega Valley Apts.	Data	12.7272727273	#6eda55
China Camp	Data	-4.72727272727	#fc7570
Delfina	Data	1.63636363636	#6eda55
East Bay Crossing	Data	-10.3636363636	#fc7570
Echo Park	Data	6.72727272727	#6eda55
Gerry Trace	Data	-2.90909090909	#fc7570
Hanger 1	Data	0.181818181818	#6eda55
Indian Valley	Data	1.45454545455	#6eda55
Joshua Tree	Data	-7.45454545455	#fc7570
Juniper	Data	-4.36363636364	#fc7570
Lower Haight	Data	7.45454545455	#6eda55
Mackinac Landing	Data	-6.90909090909	#fc7570
Marina	Data	1.45454545455	#6eda55
Market Street Apts.	Data	-4.54545454545	#fc7570
North Beach	Data	-2.54545454545	#fc7570
Papalote	Data	-1.09090909091	#fc7570
Potrero Hill	Data	-6.0	#fc7570
River Oaks	Data	0.909090909091	#6eda55
The Baltimore	Data	1.45454545455	#6eda55
The El Dorado	Data	5.09090909091	#6eda55
Travis Village	Data	-6.0	#fc7570
Viera Hills	Data	-7.81818181818	#fc7570